lib/evmone/instructions_opcodes.hpp

```diff
@@ -163,6 +163,11 @@ enum Opcode : uint8_t
     OP_DUPN = 0xb5,
     OP_SWAPN = 0xb6,
 
+    OP_DATALOAD = 0xb7,
+    OP_DATASIZE = 0xb8,
+    OP_DATACOPY = 0xb9,
+    OP_DATALOADI = 0xba,
```

Comment:
So are we going with `DATALOADI` (I for immediate) or `DATALOADN` (for being in-line with PUSHn/SWAPN/DUPN).

Reply:
I think I like `DATALOADN` now

Reply:
Let's go with `DATALOADN`. @chfast any objection?

Reply:
No.